### PR TITLE
Kan 39 fix calculating distance abnormal functionality

### PR DESCRIPTION
**Pull request by RyuJeMu**
## What

This PR includes updates related to distance calculation and display functionality. The following changes have been made:

- Fixed irregular distance calculation on double tap by including a cumulative distance variable.
- Displayed the total distance through screen readjustment.
- Managed the total distance variable within the GameScreen.

## Why

These updates enhance the user experience by ensuring accurate distance calculation and clear display of total distance, while maintaining organized management of related variables within the GameScreen.

## Screenshots
image
image
image

### CODE REVIEW

**Review by DongjunSuh923 — APPROVED** [review]
Looks good

File: src/screen/GameScreen.java [hunk 1]
```diff
@@ -404,9 +406,14 @@ public void updatePlayerDistance() {
 		int currentX = this.ship1.getPositionX();
 		int currentY = this.ship1.getPositionY();
 
+		if (ship1.getActiveDoubleTab()) {
+			lastPlaterX = currentX;
```
Comment (SEOKMINJUN) [inline src/screen/GameScreen.java:410]: Fix typo

File: src/screen/GameScreen.java [hunk 2]
```diff
@@ -404,9 +406,14 @@ public void updatePlayerDistance() {
...
+			lastPlaterY = currentY;
```
Comment (SEOKMINJUN) [inline src/screen/GameScreen.java:411]: Fix typo too

**Review by SEOKMINJUN — APPROVED** [review]
Looking Good for me except typo. Great works.

**Pull request merged**